Makefile

```diff
@@ -53,23 +53,28 @@ tools/ecfr-parser/build/ecfr-parser: tools/ecfr-parser/*.go tools/ecfr-parser/**
 
 data.prod: ## Load a Part of Title 42. e.g. make data.prod.435 will load Part 435 into prod
 data.prod: CORE_URL = https://3iok6sq3ui.execute-api.us-east-1.amazonaws.com/prod/v2/
+data.prod: SUPPLEMENTAL_URL = https://3iok6sq3ui.execute-api.us-east-1.amazonaws.com/prod/v2/supplemental_content
 
 data.val: ## Load a Part of Title 42. e.g. make data.val.435 will load Part 435 into val
 data.val: CORE_URL = https://qavc1ytrff.execute-api.us-east-1.amazonaws.com/val/v2/
+data.val: SUPPLEMENTAL_URL = https://qavc1ytrff.execute-api.us-east-1.amazonaws.com/val/v2/supplemental_content
 
 data.dev: ## Load a Part of Title 42. e.g. make dev.data.435 will load Part 435 into dev
 data.dev: CORE_URL = https://hittwbzqah.execute-api.us-east-1.amazonaws.com/dev/v2/
+data.dev: SUPPLEMENTAL_URL = https://hittwbzqah.execute-api.us-east-1.amazonaws.com/dev/v2/supplemental_content
 
 data.experimental: ## Load a Part of Title 42. e.g. make data.experimental URL=[experimental lambda URL] into dev-experimental
 data.experimental: CORE_URL = $(URL)/v2/
+data.experimental: SUPPLEMENTAL_URL = $(URL)/v2/supplemental_content
 
 data.local: ## Load a Part of Title 42. e.g. make data.local.435 will load Part 435
 data.local: CORE_URL = http://localhost:8000/v2/
+data.local: SUPPLEMENTAL_URL = http://localhost:8000/v2/supplemental_content
 data.local: export EREGS_USERNAME=RpSS01rhbx
 data.local: export EREGS_PASSWORD=UkOAsfkItN
 
 data.%: tools/ecfr-parser/build/ecfr-parser
-	./tools/ecfr-parser/build/ecfr-parser -loglevel trace -log-parse-errors=false -attempts 3 -title 42 -subchapter IV-C -parts 400,457,460 -eregs-url $(CORE_URL)
+	./tools/ecfr-parser/build/ecfr-parser -loglevel trace -log-parse-errors=false -attempts 3 -title 42 -subchapter IV-C -parts 400,457,460 -eregs-url $(CORE_URL) -eregs-supplemental-url $(SUPPLEMENTAL_URL)
```

Comment:
do we really need to pass this in?  Can't we just construct it in the app itself from the CORE_URL?  It seems relatively static.

Reply:
Wasn't the purpose of passing it in to make uploading this data optional? Could use a bool flag though